changelog.txt

```diff
@@ -6,6 +6,7 @@
 * Fix - Use of deprecated call-style to registerPaymentMethods. WooCommerce Payments now requires WooCommerce Blocks of at least version 3.9.0.
 * Fix - Deposit date on Transactions list page.
 * Add - Error message when total size of dispute evidence files uploaded goes over limit.
+* Fix - Remove duplicate text from the list inside enable currency modal when the symbol and code is the same.
```

Comment:
Well, one more exception 🙂 In this case we're working behind a feature flag for a feature that has not been released yet, so not adding changelog entries until the first release.

Reply:
Sure.